tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -1357,6 +1369,7 @@ class TfGraphControls extends LegacyElementMixin(PolymerElement) {
     this._selectedTagIndex = 0;
     this._selectedGraphType = this._getDefaultSelectionType();
     this.traceInputs = false; // Set trace input to off-state.
+    this.extractNodes = true;
```

Comment:
Optional: I would also be okay with dropping this.  Presumably, if a user is inspecting a graph that doesn't require high degree extraction, they might have several graphs of this type in their logdir.  Would it make sense to preserve the state, instead of resetting it to `true`, upon run index changed?

Reply:
I think that's more intuitive, I agree. It does look like a global UI setting, not a per-graph setting. I'll revert this reset.